server/views/pages/contacts/manage/contactSearch.njk

```diff
@@ -213,12 +213,14 @@
             {% endif %}
         </div>
 
-        <div class="govuk-grid-column-full govuk-!-margin-top-4">
-            <a  href="#" class="govuk-body" data-qa="no-result-message" id="no-result-message">
-                The contact is not listed
-            </a>
-        </div>
     {% endif %}
+     {% if results %}
```

Comment:
Fixed this as if you got no results it didn't show the link to add a new contact. 
The no results flow feels a bit janky though, will raise with Chris et-all if there is no pending tickets for it.